converter.js

```diff
@@ -30,6 +31,12 @@ export function convertFont(inputPath) {
         console.error('Error during conversion:', err);
         return;
       }
+      const ttfFile = files.find((file, i) => {
```

Comment:
Instead of finding the file, we could keep track of it earlier, but I think the possibility of an issue is low, so it's okay.

One case I thought of is: what if the user already has a matching file, or a conflicting file name for the final output too. But I think that's pretty much a corner case, so we can leave it for now.